src/qt/optionsdialog.cpp

```diff
@@ -65,6 +65,28 @@ OptionsDialog::OptionsDialog(QWidget* parent, bool enableWallet) : QDialog(paren
 #ifdef Q_OS_MAC
     /* remove Window tab on Mac */
     ui->tabWidget->removeTab(ui->tabWidget->indexOf(ui->tabWindow));
+
+
+    ui->bitcoinAtStartup->setAttribute(Qt::WA_MacShowFocusRect, 0);
+    ui->databaseCache->setAttribute(Qt::WA_MacShowFocusRect, 0);
+    ui->threadsScriptVerif->setAttribute(Qt::WA_MacShowFocusRect, 0);
+    ui->zeromintPercentage->setAttribute(Qt::WA_MacShowFocusRect, 0);
```

Comment:
Errors here, commenting out line 73 and 74 allows compiling on macOS(tested) and probably other OS's(untested)

![Screen Shot 2021-05-20 at 6 46 29 PM](https://user-images.githubusercontent.com/13294167/119069830-b7d21500-b99b-11eb-97b6-c183d47f5de6.png)

Reply:
Fixed